.github/workflows/build-and-deploy.yml

```diff
@@ -1,166 +1,138 @@
 name: Docker Build, Publish and Deploy
```

Comment:
Maybe something shorter and more flexible?
```suggestion
name: CI/CD Pipeline
```

Reply:
I'd say it's out of the scope, but since I'll have to push it again I'll do it ;)